WordPress/Classes/Utility/WPError.h

```diff
@@ -48,7 +50,7 @@
  * @param error Assumed to be an error from a networking call.
  * @returns YES if a sign-in page was shown.
  */
-+ (BOOL)showWPComSigninIfErrorIsInvalidAuth:(nonnull NSError *)error;
++ (BOOL)showWPComSigninIfErrorIsInvalidAuth:(NSError *)error;
```

Comment:
I kind of feel a null error is fine here. Otherwise something has failed and we don't show the user anything?

Reply:
I believe this one's fine because all the paths that lead to this method do not have null `error` values.